project/src/test/resources/dockerfile_assemble

```diff
@@ -14,4 +14,4 @@ WORKDIR /home/spring/server
 COPY --from=builder /proj/build/libs/server.jar ./
 USER spring
 
-ENTRYPOINT ["java", "-server", "-jar", "server.jar", "--spring.profiles.active=gcp"]
+ENTRYPOINT ["java", "-server", "-jar", "server.jar", "--spring.profiles.active=default"]
```

Comment:
If you erase the configuration file this "default" option will not have a match and it will result in an exception, right?

Reply:
No, it will fall to spring default profile